src/core/main.c

```diff
@@ -2312,6 +2325,12 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 goto finish;
 
+        if (!skip_setup && arg_early_core_pattern) {
+                r = write_string_file("/proc/sys/kernel/core_pattern", arg_early_core_pattern, 0);
+                if (r < 0)
+                        log_warning_errno(r, "Failed to init core dump pattern, ignoring: %m");
```

Comment:
Maybe just "Failed to write /proc/sys/kernel/core_pattern, ignoring: %m" ?

Reply:
maybe also include what you tried to write:

```
log_warning_errno(r, "Failed to write '%s' to/proc/sys/kernel/core_pattern, ignoring: %m", arg_early_core_pattern);
```

Reply:
OK